lib/perl/Genome/Model/CwlPipeline/Command/Run.pm

```diff
@@ -332,6 +333,7 @@ sub _stage_cromwell_output {
         return 1;
     }
 
+    return unless ref $item eq 'HASH';
```

Comment:
Is there ever a time when we'd want to persist these "simple" outputs in a file or somesuch? I guess without context like a label or something, a random number isn't all that useful..

Reply:
Interesting question!  They're all in the Cromwell database (where we got this data we're sifting through), so if we decide we do want to do that it's not too late to go back and backfill later 😄